src/community/ogcapi/ogcapi-coverages/src/test/java/org/geoserver/ogcapi/v1/coverages/CollectionsTest.java

```diff
@@ -75,7 +75,7 @@ public void revertChanges() throws IOException {
         getCatalog().save(c);
     }
 
-    @SuppressWarnings("unchecked") // generics varargs creation by hamcrest
+    // generics varargs creation by hamcrest
```

Comment:
There are a bunch of comments that are present (like this one, or "matcher varargs" or "EMF mode without generics". Those feel like justification for suppressions that have been removed, and would probably be cleaner if they were removed.

Not really affecting anything in terms of tests, just not that useful.

Reply:
Ah... I relied on IntelliJ to perform this cleanups, of course it did not recognize the meaning of the comments. Cleaning it up.